client/src/components/UserContacatCard/UserContactCard.js

```diff
@@ -0,0 +1,73 @@
+import {
+  Card,
+  CardContent,
+  Typography,
+  Avatar,
+  colors,
+
+} from '@mui/material';
+import MdPhone from '@mui/icons-material/Phone';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import EmailIcon from '@mui/icons-material/Email';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles({
+  card: {
+    backgroundColor: 'secondary',
+    maxWidth: 345,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: '10px',
+  },
+  icon: {
+    marginRight: '0.5rem',
+  },
+});
+const contentStyles = { display: 'flex', marginBottom: '15px', alignItems: 'center' };
+function userContactCard({
```

Comment:
You should start the name of the component capital  letter like this   UserContactCar